mypy.ini

```diff
@@ -13,6 +13,9 @@ disallow_any_unimported = True
 disallow_subclassing_any = True
 no_implicit_optional = True
 
+# Some functions in e3.log do override without preserving full signature
+disable_error_code = override
```

Comment:
I would prefer accepting my PR instead https://github.com/AdaCore/e3-core/pull/542 in order to keep that check where it makes sense for us.